<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionEditDialog</class>
 <widget class="QDialog" name="ConnectionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vl_paramconn">
     <item>
      <widget class="QLabel" name="l_paramconn">
       <property name="text">
        <string>Параметры соединения</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_paramconn">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="l_nameconn">
           <property name="text">
            <string>Имя подключения</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_nameconn"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Тип базы данных</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_type">
           <item>
            <property name="text">
             <string>QSQLITE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QMYSQL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QPSQL</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="l_server">
           <property name="text">
            <string>Сервер</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_server"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="l_login">
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_login"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="l_port">
           <property name="text">
            <string>Порт</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="sb_port">
           <property name="maximum">
            <number>8000</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="l_password">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="le_password">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vl_filedb">
       <item>
        <widget class="QLabel" name="l_chosefile">
         <property name="text">
          <string>Выберите файл из БД</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_filedb"/>
         </item>
         <item>
          <widget class="QToolButton" name="btn_connect">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ok">
           <property name="text">
            <string>Принять</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
